cmd/kogito/command/install/kogito_operator.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/shared"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/operator"
+	"strings"
+
+	"github.com/gobuffalo/packr/v2"
+
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/meta"
+	"github.com/kiegroup/kogito-cloud-operator/version"
+
+	"github.com/spf13/cobra"
+
+	apps "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+const (
+	defaultOperatorImageName   = "quay.io/kiegroup/kogito-cloud-operator"
+	boxDeployPath              = "../../../../deploy"
+	fileOperatorYaml           = "operator.yaml"
+	fileRoleYaml               = "role.yaml"
+	fileRoleBindingYaml        = "role_binding.yaml"
+	fileServiceAccountYaml     = "service_account.yaml"
+	fileKogitoAppCRDYaml       = "crds/app_v1alpha1_kogitoapp_crd.yaml"
+	fileKogitoDataIndexCRDYaml = "crds/app_v1alpha1_kogitodataindex_crd.yaml"
+)
+
+var (
+	operatorImageName = fmt.Sprintf("%s:%s", defaultOperatorImageName, version.Version)
+)
+
+type installKogitoOperatorFlags struct {
+	Namespace string
+	Image     string
+}
+
+type installKogitoOperatorCommand struct {
+	context.CommandContext
+	flags   installKogitoOperatorFlags
+	command *cobra.Command
+	Parent  *cobra.Command
+}
+
+func newInstallKogitoOperatorCommand(ctx *context.CommandContext, parent *cobra.Command) context.KogitoCommand {
+	command := installKogitoOperatorCommand{
+		CommandContext: *ctx,
+		Parent:         parent,
+	}
+
+	command.RegisterHook()
+	command.InitHook()
+
+	return &command
+}
+
+func (i *installKogitoOperatorCommand) Command() *cobra.Command {
+	return i.command
+}
+
+func (i *installKogitoOperatorCommand) RegisterHook() {
+	i.command = &cobra.Command{
+		Use:     "operator [flags]",
+		Short:   "Installs the Kogito Operator into the OpenShift/Kubernetes cluster",
+		Example: "install operator -p my-project",
+		Long: `Installs the Kogito Operator via OLM if available by subscripting to the Kogito Operator.
```

Comment:
@ricardozanini  Does operator subscribe for Kogito Operator in case it is available in operatorhub? From the mail you sent I understood that it will just inform user and let him install  the operator.

Reply:
No, it CLI won't create an automatic subscription. And YES, I forgot to change the message. :laughing: